### ISSUE TITLE
[a11y] Convert Coordinates in the WB does not work

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1495]
![image](https://user-images.githubusercontent.com/37256050/167680167-4180a072-ed1c-44b6-8bff-8c7eb0fab1d1.png)

It does nothing

db: `oldfish`
user: `abentley`
`/specify/workbench/2/`
